src/main/scala/io/iohk/ethereum/blockchain/sync/fast/SyncMetrics.scala

```diff
@@ -1,28 +1,39 @@
 package io.iohk.ethereum.blockchain.sync.fast
 
 import java.util.concurrent.atomic.AtomicLong
-
 import com.google.common.util.concurrent.AtomicDouble
 import io.iohk.ethereum.blockchain.sync.fast.FastSync.SyncState
 import io.iohk.ethereum.metrics.MetricsContainer
 
+import scala.concurrent.duration.MILLISECONDS
+
 object SyncMetrics extends MetricsContainer {
 
-  private[this] final val PivotBlockNumberGauge =
+  private final val PivotBlockNumberGauge =
```

Comment:
shall we keep private[this]? it imposes a bigger restriction

Reply:
As far as I understand `private[this]` is not more restrictive. No place outside this object can access that val already, or tries to

Reply:
[private] makes it available to the current instance and other instances of the class it’s declared in. Since we are already using a singleton - there are no other instances that could access it. So there's no difference in this case. Some consider it a good practice to default to the highest restriction, I guess this was the reason private[this] was used here on the first place. We could make a decision as a team on which guideline to follow